src/Lambda/Input/PublishLayerVersionRequest.php

```diff
@@ -125,7 +125,7 @@ public function requestBody(): string
 
         (static function (array $input) use (&$payload) {
             foreach ($input as $value) {
-                $payload['CompatibleRuntimes'][] = $value;
+                $payload['CompatibleRuntimes']['member'][] = $value;
```

Comment:
This has been changed by my fix. Can you confirm?

Reply:
Yes. It looks correct. 

I checked with the specification.